sql/timeseries/asyncClipboardRead.sql

```diff
@@ -0,0 +1,18 @@
+#standardSQL
+SELECT
+  SUBSTR(_TABLE_SUFFIX, 0, 10) AS date,
+  UNIX_DATE(CAST(REPLACE(SUBSTR(_TABLE_SUFFIX, 0, 10), '_', '-') AS DATE)) * 1000 * 60 * 60 * 24 AS timestamp,
+  IF(ENDS_WITH(_TABLE_SUFFIX, 'desktop'), 'desktop', 'mobile') AS client,
+  ROUND(SUM(IF(IFNULL(
+    JSON_EXTRACT(payload, '$._blinkFeatureFirstUsed.Features.AsyncClipboardAPIRead'),
+    JSON_EXTRACT(payload, '$._blinkFeatureFirstUsed.Features.2369'))
```

Comment:
We could query only the blink features by using the `httparchive.blink_features` table directly. For example:

```sql
SELECT * FROM `httparchive.blink_features.usage` WHERE id = '2369'
```

This table is very small, so much more efficient. Could you update the queries like this one to use the `blink_features` table?

Reply:
Also consider including `num_urls` in the results. Even though the `percent` value is graphed, many metrics' percents are 0.0% so it may be helpful to drill down into absolute counts in the data table:

![image](https://user-images.githubusercontent.com/1120896/88328790-f30da400-ccf6-11ea-9f46-a98f184ab323.png)